test/Microsoft.AspNetCore.Authentication.Test/TwitterTests.cs

```diff
@@ -18,6 +18,19 @@ namespace Microsoft.AspNetCore.Authentication.Twitter
     public class TwitterTests
     {
         [Fact]
+        public async Task VerifySignInSchemeCannotBeSetToSelf()
+        {
+            var server = CreateServer(o =>
+            {
+                o.ConsumerKey = "Test Consumer Key";
+                o.ConsumerSecret = "Test Consumer Secret";
+                o.SignInScheme = TwitterDefaults.AuthenticationScheme;
```

Comment:
Any tests for users specifying DefaultSignInScheme wrong rather than the direct option?

Reply:
I added 2 variants to the FB tests to verify the default sign in / default scheme flow, facebook's CreateServer was the ugliest so it exposed the proper overloads needed :)